### PR TITLE
Create CITATION.cff

### PR DESCRIPTION
**Pull request by jspaaks**
Added Citation metadata and teh zenodo equivalent

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "creators": [
+        {
+            "affiliation": "Netherlands eScience Center",
+            "name": "Renaud, Nicolas",
```
Comment (junghans) [inline .zenodo.json:5]: I guess more people need to be added here....

Comment (jspaaks) [reply .zenodo.json:5]: Indeed this was just an example I was doing with Nico

**Comment by JensWehner** [comment]
@jspaaks who is working on this now?

**Comment by jspaaks** [comment]
> @jspaaks who is working on this now?

Not me, feel free to edit as you like

**Comment by junghans** [comment]
Replaced by https://github.com/votca/votca/pull/845

**Pull request closed without merging**